dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -28,6 +28,18 @@ def replace_hostname(url)
   url.gsub(/(\w+)\.(\w+)\.code\.org/,'\1-\2.code.org')
 end
 
+def color_for_status(status)
+  {
+    submitted: 'rgb(118, 101, 160)',
```

Comment:
no way to get the colour values out of the original color.scss?  i guess this is a good double-check, we'll just have to update here if we change colours.